src/js/core/navbar.js

```diff
@@ -162,6 +162,9 @@ export default {
             },
 
             handler(_, {$el, dir}) {
+                if (!hasClass($el, this.clsDrop)) {
```

Comment:
Instead of checking for clsDrop, I think it would be better to use the `getDropdown` function and check the $el's parents. clsDrop is only used for auto creating the dropdowns.

Reply:
Hmmm.. I'm not quite sure what you mean. What's the purpose of getting the dropdown component of the current `$el`? And what for and where do you want to check the parents?

Reply:
You are right, only the top level dropdowns should have the `clsDrop` class.